integration-libs/cdc/root/service/cdc-js.service.ts

```diff
@@ -162,6 +167,146 @@ export class CdcJsService implements OnDestroy {
     }
   }
 
+  /**
+   * Trigger CDC User registration and log in using CDC APIs.
+   *
+   * @param email
+   * @param password
+   */
+  registerUserWithoutScreenSet(user: UserSignUp) {
+    if (user.uid && user.password) {
+      (this.winRef.nativeWindow as { [key: string]: any })?.[
+        'gigya'
+      ]?.accounts?.initRegistration({
+        callback: (response: any) => {
+          this.zone.run(() => this.onInitRegistrationHandler(user, response));
+        },
+      });
+    }
+  }
+
+  /**
+   * Trigger CDC User registration using CDC APIs.
+   *
+   * @param response
+   */
+  onInitRegistrationHandler(user: UserSignUp, response: any) {
```

Comment:
nitpick: I think we can make this method `protected`, as it's not meant to be called from outside. Same for other helper/handler methods in this PR.

Reply:
sure Kris, will do